src/pybind/ceph_argparse.py

```diff
@@ -181,7 +181,7 @@ def __str__(self):
         if len(self.range) == 2:
             r = '[{0}-{1}]'.format(self.range[0], self.range[1])
 
-        return '<int{0}>'.format(r)
+        return '(int{0})'.format(r)
```

Comment:
Why the switch from <> to () for type?  I originally thought it was a useful distinction to indicate "placeholder with type" rather than just "formal parameter name"

Reply:
The main shift with this PR is that instead of just naming the type of argument, we usage shows the name of the field.  So for example
```
osd set-full-ratio <float[0.0-1.0]>                                set usage ratio at which OSDs are marked full
```
becomes
```
osd set-full-ratio <ratio(float[0.0-1.0])>                                set usage ratio at which OSDs are marked full
```
A better example is maybe
```
osd reweight-by-utilization [<oload(int)>] [<max_change(float)>] [<max_   reweight OSDs by utilization [overload-percentage-for-consideration, 
 osds(int)>] [--no-increasing]                                             default 120]
```
which previous would just show you that there was an int, float, and then int argument, but not what the arguments are named or mean.

Once you have the ``<name>`` there, then adding type needs something other than angle braces.  An alternative to parens might be ``<name:type>``?

Reply:
Okay, I see.  I guess I'm searching my brain for existing practice and not really coming up with many other examples of describing a placeholder for param-with-type.  *I* personally think of <> as a "type", not a name, so might have been tempted to go with {ratio<float[0.0-1.0]>} to indicate that it's a param, required, and set off the type with <>.

Reply:
(and with the second example, I think [oload<int>] [max_change<float>] works.)  But I'm not adamant and this is largely preference.

Reply:
Yeah, I'm thinking that the int, float, and even pgid typenames can be left off since they're usually clear from context or the field name.

The main net of all of this then is that ``{myarg}`` becomes ``<myarg>``, which is more typical (e.g., in man pages or python argparse help).

Reply:
sigh.  markup chewed up the intent of my comment

Reply:
and I don't think that int and float are always clear from context, especially float